README.md

```diff
@@ -22,143 +20,72 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/mineiros-io/terramate)](https://goreportcard.com/report/github.com/mineiros-io/terramate)
 [![codecov](https://codecov.io/gh/mineiros-io/terramate/branch/main/graph/badge.svg?token=gMRUkVUAQ4)](https://codecov.io/gh/mineiros-io/terramate)
 
-Terramate is a tool for managing multiple terraform stacks.
+Terramate is a tool for managing multiple Terraform stacks.
+It provides ways to keep your code [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)
+and also manage dependencies between stacks with minimal effort to get
+started in a non-intrusive way.
 
-The stack concept is not defined by Hashicorp's Terraform tooling but just a
```

Comment:
I was trying to make the docs more direct and was thinking that it is common knowledge that Terraform doesnt have actual stacks...because they dont :laughing: (unless the person doens't know Terraform very well, but I dont think it is the case with the target audience). WDYT @soerenmartius ?

Reply:
I would leave this in :)

Reply:
Added it back